### DISCUSSION TITLE
在kaggle上安装不上。

### DISCUSSION THREAD

**Discussion by stoneken18** [PaddlePaddle/PaddleHelix#277] [General]
我用以下命令在kaggle平台安装总出问题，在别的机器也出一样的问题：
!pip install paddlepaddle-gpu -i https://mirror.baidu.com/pypi/simple
!pip install rdkit-pypi
!pip install pgl
!pip install paddlehelix
import paddlehelix as padhelix

错误：
ERROR: Could not find a version that satisfies the requirement paddlehelix (from versions: none)
ERROR: No matching distribution found for paddlehelix

**Comment by stoneken18** [comment]
我刚试，怎么在自家的平台Baidu AI studio都装不了？？？

**Reply by Noisyntrain** [reply]
Hi stoneken18, 请问您具体想使用paddlehelix的哪个功能，可以在apps目录下根据您想要功能的Readme来安装依赖和运行代码。 或者您可以参考https://aistudio.baidu.com/projectdetail/1293361?contributionType=1 在aistudio上运行相应的项目，希望能够对您有所帮助

**Reply by stoneken18** [reply]
您好，我是想用paddlefsl作小样本图像分类，结果这个依赖paddlehelix. 按照网上你们给出的各种instruction在哪都装不了这个paddlehelix, 永远都是这个错误：
ERROR: Could not find a version that satisfies the requirement paddlehelix (from versions: none)
ERROR: No matching distribution found for paddlehelix。 

你们的paddle 套件开发的网上看起来挺好，怎么感觉都不兼容其它机器平台，想试用一下，除了最基础的paddlepaddle凑活能装，其它套件好像都装不了。都快失去对paddlepaddle的兴趣了。

**Reply by Noisyntrain** [reply]
了解了。paddlehelix是一个主要在生物化学领域应用的工具集，我理解不安装Paddlehelix应该不会影响paddlefsl的正常使用？您可以试试不安装paddlehelix的情况下运行代码看看。另外，请问您看到的paddlefsl对paddlehelix依赖的文档有具体网址么，希望能够帮您解决问题

**Reply by stoneken18** [reply]
谢谢，装完paddlefsl, import paddlefsl,就会提示要paddlehelix.

**Reply by Noisyntrain** [reply]
好的，我明白了。我在aistudio平台测试了下使用命令：python -m pip install paddlehelix 可以顺利安装paddlehelix并且顺利import
![image](https://github.com/PaddlePaddle/PaddleHelix/assets/44316368/0a91b06e-57c8-4ce9-ac6f-ce6a1f30c156)
，请问您使用的是什么命令安装的paddlehelix？以及安装平台的环境是什么样的（python版本，操作系统信息）

**Comment by stoneken18** [comment]
我在好几个平台试过都不行，另外就是最基础的paddlepaddle 与pytorch比，兼容性也不好，有些也装不了，或装上用不了cuda,希望你们好好把安装的兼容性改进一下，多测试不同的平台，要不像我这样本来很想试试国产的，现在兴趣也不大了，谢谢！
